### DISCUSSION TITLE
why did i can't gacha 为什么我不能抽卡

### DISCUSSION THREAD

**Discussion by happy233333** [Melledy/Grasscutter#1425] [General]
when i try to gacha, I will be prompted with an internal error in the server
当我准备抽卡时就会提示服务器内部错误

Grasscutter version: v1.2.0
screenshots 截图：
![image](https://user-images.githubusercontent.com/108604530/177027286-41e75515-3e63-4266-82ac-16b440fe3391.png)
![image](https://user-images.githubusercontent.com/108604530/177027344-cf7d1a1d-6a51-4a45-a033-67f2151fc7bd.png)

what's wrong with it?

**Comment by zhaodice** [comment]
背包满了

**Reply by happy233333** [reply]
但是提示的是服务器内部错误，而不是背包已满

**Reply by zhaodice** [reply]
> 但是提示的是服务器内部错误，而不是背包已满

那是因为割草机还没实现提示背包满的功能，出现这个情况就是背包满了，没有其他可能。

**Reply by Birdulon** [reply]
> 那是因为割草机还没实现提示背包满的功能

We tried our best :(
<img width="1061" alt="image" src="https://user-images.githubusercontent.com/5397662/177137932-51d8846a-f861-4f81-a0c0-68d97f386eeb.png">

**Reply by Tesutarin** [reply]
> 那是因为割草机还没实现提示背包满的功能

他用的这个版本太老了，之前就把内部错误改了 #1262
